plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py

```diff
@@ -73,6 +76,136 @@
 }
 
 
+class DataEncrypt:
+    """Class to encrypt/decrypt data strings with password provided."""
+
+    hash_algo = "sha256"
+    hash_iterations = 20000
```

Comment:
isn't better to:

- add `hash_algo` and `hash_iterations` as parameters of the main APIs below
- make those parameters optional, with default values `_DEFAULT_HASH_ALGO` and `_HASH_ITERATIONS` respectively
- rename these class variables with `_DEFAULT_HASH_ALGO` and `_HASH_ITERATIONS`

Reply:
otherwise, say I wanted to use a different algorithm (either online or offline); I would have to modify these class variables which could break other code.

Reply:
@solarw ^.